### ISSUE TITLE
Customizable reports not reading at startup.

### ISSUE THREAD

**Issue by smalers** [OpenWaterFoundation/owf-tstool-timesheetscom-plugin#15]
The customizable report data that are read for project data are not returning any data, as per the following error.  The `ProjectRecordBillableStatus` is set so maybe these are now in separate fields?

```
Status[2](TimesheetsComDataStore.readReportProjectCustomizable): Reading report project customizable from: https://secure05v.timesheets.com/api/public/v1/report/project/customizable?StartDate=2019-01-01&EndDate=2024-12-31&AllAccountCodes=1&AllCustomers=1&AllUsers=1&AllProjects=1&Approved=RECORD_UNAPPROVED,RECORD_APPROVED&GroupType=Project&IncludeWorkDescription=1&ProjectRecordBillableStatus=RECORD_BILLABLE,RECORD_UNBILLABLE&ProjectRecordStatus=PROJECTRECORDSTATUS_ALL&ReportType=Detailed&Signed=RECORD_UNSIGNED,RECORD_SIGNED
Status[2](JacksonToolkit.logJsonTree): {
  "errors" : [ "Billable status selection required.", "Record status selection required." ],
  "processing" : {
    "writetostorage" : "false",
    "queued" : "false",
    "reportuuid" : ""
  },
  "report" : { }
}
```

**Comment by smalers** [comment]
Based on the documentation ([https://secure05v.timesheets.com/api/public/v1/index.cfm?docs](https://secure05v.timesheets.com/api/public/v1/index.cfm?docs)), I changed to `Billable` and `RecordStatus`, which works:

```
Status[2](TimesheetsComDataStore.readReportProjectCustomizable): Reading report project customizable from: https://secure05v.timesheets.com/api/public/v1/report/project/customizable?StartDate=2019-01-01&EndDate=2024-12-31&AllAccountCodes=1&AllCustomers=1&AllUsers=1&AllProjects=1&Approved=RECORD_UNAPPROVED,RECORD_APPROVED&GroupType=Project&IncludeWorkDescription=1&Billable=RECORD_BILLABLE,RECORD_UNBILLABLE&RecordStatus=PROJECTRECORDSTATUS_ALL&ReportType=Detailed&Signed=RECORD_UNSIGNED,RECORD_SIGNED
Status[2](JacksonToolkit.logJsonTree): {
  "errors" : [ ],
  "processing" : {
    "writetostorage" : "false",
    "queued" : "false",
    "reportuuid" : ""
  },
  "report" : {
... data follows...
```

**Comment by smalers** [comment]
I updated the code and released as plugin version 1.1.4.  Close this issue.
